codegen/JsonUtils/src/main/java/ConstantValue.java

```diff
@@ -0,0 +1,52 @@
+import java.text.DateFormat;
+import java.util.Date;
+
+public class ConstantValue implements Value {
+    private final String columnName;
+    private final Object value;
+    private final Class type;
+
+    public Object getValue() {
+        return value;
+    }
+
+    public Class getType() {
+        return type;
+    }
+
+    public static ConstantValue newInstance(final String val, final String type, final String columnName) throws Exception {
+        CheckerUtils.checkNullOrEmpty(val, "val");
+        CheckerUtils.checkNullOrEmpty(type, "type");
+        CheckerUtils.checkNullOrEmpty(columnName, "columnName");
+
+        String typeLower = type.toLowerCase();
+        Object value;
+        Class clss = Type.getClass(typeLower);
+        if (clss == null) {
+            throw new RuntimeException("Unknown data type: " + type);
+        }
+
+        if (clss.equals(Date.class)) {
+            DateFormat date_format = new java.text.SimpleDateFormat("yyyy-MM-dd");
```

Comment:
The data_format is better to be a global variable and has only one instance, otherwise will be a major performance drawback.

Reply:
done